### DISCUSSION TITLE
osu!lazer won't run

### DISCUSSION THREAD

**Discussion by Brxby** [ppy/osu#23351] [Q&A] [answered]
I have stable installed and it works perfectly, but when I try to run lazer nothing happens.
Here are the event data and system data from the event logs (I'll attach a screenshot for you to read it better :)

System

Provider
[ Name] Application Error

EventID 1000
[ Qualifiers] 0

Version 0

Level 2

Task 100

Opcode 0

Keywords 0x80000000000000

TimeCreated
[ SystemTime] 2023-04-30T17:33:47.4858715Z

EventRecordID 2242387

Correlation

Execution
[ ProcessID] 0
[ ThreadID] 0

Channel Application

Security

EventData

osu!.exe
0.0.0.0
63ed8286
EZFRD64.DLL
0.0.0.0
4693218c
c0000005
0000000000003654
25f4
01d97b89eac02bcf
C:\Users\Moni\AppData\Local\osulazer\app-2023.403.0\osu!.exe
C:\Windows\USB Vibration\7906\EZFRD64.DLL
6f488287-41a7-48bb-b68f-542c02e46203
![Screenshot 2023-04-30 204648](https://user-images.githubusercontent.com/120403978/235368436-1cc964bf-5adf-4cee-9a03-83beac532ce3.png)
![Screenshot 2023-04-30 204604](https://user-images.githubusercontent.com/120403978/235368440-4613de8e-510e-4fdc-be45-59e273804f4c.png)

**Comment by bdach** [comment, marked as answer]
See https://github.com/ppy/osu/issues/13634

Caused by a buggy controller driver. I recommend removing it for now.
